### PR TITLE
Use PF4 Switch

### PR DESCRIPTION
**Pull request by rszwajko**
Replace custom wrapper around PF3 with PF4 Switch component.

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by rszwajko** [comment]
switch_b
switch_a

**Comment by sjd78** [comment]
ci add to whitelist

**Comment by sjd78** [comment]
ci test

**Comment by sjd78** [comment]
> switch_a

I made a few layout changes in #1381 that should make the label and switch line up better.

**Comment by sjd78** [comment]
ci test

File: yarn.lock [hunk 1]
```diff
@@ -8043,7 +7874,12 @@ lodash.words@^3.0.0:
   dependencies:
     lodash._root "^3.0.0"
 
-lodash@4.17.21, lodash@^4.13.1, lodash@^4.14.2, lodash@^4.17.10, lodash@^4.17.11, lodash@^4.17.12, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.2, lodash@^4.17.4, lodash@^4.17.5, lodash@^4.2.0, lodash@^4.3.0:
+lodash@4.17.15:
```
Comment (sgratch) [inline yarn.lock:7877]: same comment here: https://github.com/oVirt/ovirt-web-ui/pull/1427#discussion_r639812744

**Review by sgratch — COMMENTED** [review]
LGTM except the lodash version issue.

File: yarn.lock (hunk 1 above)
Comment (rszwajko) [reply yarn.lock:7877]: Fixed in #1437

**Review by sgratch — APPROVED** [review]
LGTM

**Comment by isaranova** [comment]
LGTM

**Pull request merged**